### DISCUSSION TITLE
support ssml tags

### DISCUSSION THREAD

**Discussion by omega3** [k2-fsa/sherpa-onnx#519] [Ideas]
Are you going to support ssml tags? Example:
Supported SSML Tags - Amazon Polly
https://docs.aws.amazon.com/polly/latest/dg/supportedtags.html#break-tag

At least pauses between sentences would be a huge difference.

On top of that, a lot of pre-trained models speak very fast and because of this don't sound natural. There are sometimes even no distinct breaks between words (for example vctk models available for Piper). I haven't checked if you use this models. They sound quite natural but are so fast that are almost unusable (at least in some cases).

**Comment by csukuangfj** [comment]
Could you describe how users would use it if they don't know what ssml tags are?

Currently, users only need to enter normal text to get audio samples.

**Reply by woheller69** [reply]
it would be great if ssml could be supported or stripped away, see:
https://github.com/woheller69/ttsEngine/issues/115

**Comment by omega3** [comment]
You can write in project description about it. 
SSML tags are nothing unusual. Google also uses them:
https://cloud.google.com/text-to-speech/docs/ssml
https://cloud.google.com/text-to-speech/#section-2
